### DISCUSSION TITLE
Show parent folders in top bar settings, keep order of presets in styles.csv (like before)

### DISCUSSION THREAD

**Discussion by finefin** [vladmandic/sdnext#2304] [General]
Hi, I have a lot of different checkpoints, lora and embeddings that I store on an external drive. They are sorted in folders by type (dreambooth, styles, nsfw etc) - I added the base folder to the system paths and the models can be loaded just fine.

BUT a few updates ago the "base model" and "styles" lists both changed from displaying the parent folder and keeping the original order of the styles.csv to pure alphabetical order. 

Unfortunately this breaks my workflow and now I always open a file browser, look for the model that I want and then search it in the webui. Also all my style presets are now sorted alphabetically which breaks my well sorted style list.

Is there a way to switch back to the original behaviour? I can't find anything in the settings :(

**Comment by vladmandic** [comment]
this has been completely redesigned in dev branch. you can check it out or wait couple of days until its merged to master.

**Reply by finefin** [reply]
awesome. thank you!
